### ISSUE TITLE
Incorrect training loss display in progress bar

### ISSUE THREAD

**Issue by ehooon** [ParaMind2025/CAN#6]
Hi! Thanks for sharing the code. 

I noticed a minor bug in `train_one_epoch` regarding how the training loss is calculated for the `tqdm` progress bar.

**The Issue:**
```python
running_loss += loss.item()
# ...
total += labels.size(0)
pbar.set_postfix(loss=f"{running_loss/total:.4f}", acc=f"{100.*correct/total:.2f}%")
```
Since `nn.CrossEntropyLoss` returns the batch mean by default, `running_loss` accumulates batch averages. Dividing this by the total number of samples (`total`) scales the displayed loss down incorrectly.

*(Note: This is purely a display issue. It does not affect `loss.backward()` or the accuracy calculation at all.)*

**The Fix:**
Simply multiply `loss.item()` by the batch size before accumulating:

```python
# Change this:
# running_loss += loss.item()

# To this:
running_loss += loss.item() * inputs.size(0)
```

Hope this helps! Let me know if you'd like a quick PR for this.
